src/assets/scripts/game/GameClockView.js

```diff
@@ -0,0 +1,118 @@
+import { digits_integer } from '../utilities/radioUtilities';
+import { SELECTORS } from '../constants/selectors';
+import { TIME } from '../constants/globalConstants';
+
+/**
+ * Manages a clock that stays in sync with the current game time
+ * @class GameClockView
+ */
+export default class GameClockView {
+    /**
+     * @for GameClockView
+     * @constructor
+     */
+    constructor($element) {
+        this.$element = $element;
+        this.startTime = 0;
+        this.time = 0;
+
+        return this._init($element);
+    }
+
+    /**
+    * Get current time in the user's time zone
+    * @for GameClockView
+    * @property realWorldCurrentLocalTime
+    * @return {number} ms since 01/01/1970, 00:00:00 (user's time zone)
+    */
+    get realWorldCurrentLocalTime() {
+        return new Date().getTime();
+    }
+
+    /**
+    * Get current zulu time in milliseconds
+    * @for GameClockView
+    * @property realWorldCurrentZuluTime
+    * @return {number} ms since 01/01/1970, 00:00:00 UTC
```

Comment:
if you're returning a variable, be sure to add that to the doc block too:
`@return utc {number}`